server_addon/houdini/client/ayon_houdini/version.py

```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+"""Package declaring Houdini addon version."""
+__version__ = "0.3.14"
```

Comment:
```suggestion
__version__ = "0.3.0"
```

Reply:
I think I resolved the conflict when merging the develop branch in a wrong way!
Thanks.

Reply:
fixed in https://github.com/ynput/ayon-core/pull/516/commits/a34647b17545bec61bab87ba7a2097c17b0b059f